PyOpenWorm/context.py

```diff
@@ -203,6 +190,7 @@ def save_context(self, graph=None, inline_imports=False, autocommit=True, seen=N
 
         if autocommit and hasattr(graph, 'commit'):
             graph.commit()
+        return graph
```

Comment:
If you add this, then don't forget to update the doc tests in README.md

Reply:
It's fine. I didn't have plans one way or the other.

Reply:
Ok, so I think "less code is better", I removed it and update tests.